kotlin/src/test/kotlin/io/smallrye/mutiny/coroutines/DeferredAsUniTest.kt

```diff
@@ -1,16 +1,14 @@
 package io.smallrye.mutiny.coroutines
 
 import io.smallrye.mutiny.helpers.test.UniAssertSubscriber
-import kotlinx.coroutines.CancellationException
-import kotlinx.coroutines.ExperimentalCoroutinesApi
-import kotlinx.coroutines.GlobalScope
-import kotlinx.coroutines.async
-import kotlinx.coroutines.delay
-import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.*
```

Comment:
please stay with single named imports.

Reply:
New laptop; I forgot that IntelliJ 'helpfully' did this. :)